### ISSUE TITLE
Is the file compressed again if it's renamed?

### ISSUE THREAD

**Issue by kayg04** [ckt1031/obsidian-tinypng-plugin#50]
Hi, Since there's only one command that compresses all images in the vault, I often rename my images through the Attachment Management plugin, would doing so make this plugin compress the images again?

Thanks!

**Comment by kayg04** [comment]
Apparently yes, I have been compressing my images multiple times. :/

**Comment by ckt1031** [comment]
Yes, it matches the filename to check the status of compression. I would like to see if I can implement the hashing feature to ensure no images **are** compressed **multiple** times.

**Comment by ckt1031** [comment]
Improved in `0.3.0`, sorry for the huge delay.
